charts/spire/charts/spire-agent/templates/daemonset.yaml

```diff
@@ -71,6 +71,10 @@ spec:
           image: {{ template "spire-lib.image" (dict "appVersion" $.Chart.AppVersion "image" .Values.image "global" .Values.global) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args: ["-config", "/run/spire/config/agent.conf"]
+          {{- if gt (len .Values.extraEnvVars) 0 }}
```

Comment:
```suggestion
          {{- with .Values.extraEnvVars }}
```

Reply:
yeah, consistency within the charts has not been kept very well. Trying to clean it up as we go.

Reply:
I'm ok skipping this part for now too though.